restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -183,7 +183,7 @@ public static BuiltCondition idToWhere(String val, String name, Table tableData)
       value = toCqlValue(type, val);
     }
 
-    return BuiltCondition.of(name.toLowerCase(), Predicate.EQ, value);
+    return BuiltCondition.of(name, Predicate.EQ, value);
```

Comment:
Can we add some integration tests for this?

Reply:
Furthermore, is this backward compatible or would we break somebody? Also, are there other places where `.toLowerCase()` is used?

Reply:
I searched and PR fixes two places where `.toLowerCase()` is used.

I added an integration test that tests creating and accessing tables with mixed case table name and column names.